src/components/ui/CreateFormDropdown.tsx

```diff
@@ -18,7 +20,7 @@ export default function CreateFormDropdown() {
   
 
 
-  const onClickAddForm = async (type: FormType) => {
+  const onClickAddForm = async (type: NextFormType) => {
     try {
       const nextFormTitle = getNextFormTitle(type);
```

Comment:
Is it `getNextFormTitle` or `CreateNextFormTitle`?

Reply:
sure, CreateNextFormTitle sounds better